lib/ExpensiMark.js

```diff
@@ -267,7 +267,7 @@ export default class ExpensiMark {
             },
             {
                 name: 'anchor',
-                regex: /<(a)[^><]*href\s*=\s*(['"])(.*?)\2(?:".*?"|'.*?'|[^'"><])*>(.*?)<\/\1>(?![^<]*(<\/pre>|<\/code>))/gi,
+                regex: /<(a)[^><]*href\s*=\s*(['"])(.*?)\2(?:".*?"|'.*?'|[^'"><])*>(.*?)<\/\1>(?![^<]*(<\/pre>|<\/code>))/gis,
```

Comment:
I recommend we use `\s\S` and it is uniform with our usage.

Reply:
Sorry about my confusion